detailed.html

```diff
@@ -416,7 +416,7 @@ <h1 class="h3 mb-0 font-weight-bold text-gray-800">Performance Detailed View</h1
                             <div class="card shadow mb-4">
                                 <!-- Card Header -->
                                 <div class="card-header py-3 d-flex flex-row align-items-center justify-content-between">
-                                    <a class="m-0 font-weight-bold text-dark" href="detailed.html#rpslatency">Requests per Second Latency (μs)</a>
+                                    <a class="m-0 font-weight-bold text-dark" href="detailed.html#rpslatency">Requests per Second P90 Latency (μs)</a>
```

Comment:
I want to keep RPS. I want to add latency.

Reply:
I am not removing RPS. I am replacing RPS latency with per-commit P90 RPS latency. The original RPS latency graph which is only for the latest commit doesn't seem very useful. I can add P99, P999 and P50 if you want, which is basically (sort of) a super set of the original latency graph. What do you think?

Reply:
I still want to be able to see the latest-commit RPS latency. I do find it useful to see how the curves compare between the plaftorms.

Reply:
Ok, I will add it back.

Reply:
Done